utils/convert-illustration.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Validate command line arguments
+if [ ! $# -eq 2 ]
+then
+    echo "Error: Incorrect number of parameters (exactly 2 parameters required)."
+    echo "Usage: "$0" <source/publication.svg> <destination/publication.svg>"
+    exit 1
+fi
+
+source utils/convert-svg.sh
+
+# Exit when any command fails
+set -e
+
+# Set up file names
+SRC_FILE_SVG=$1
+DST_FILE_SVG=$2
+
+# Create destination folders if necessary
+mkdir -p $(dirname $DST_FILE_SVG)
+
+# Convert SVG into PNG of width 1200px.
+echo -e `basename $SRC_FILE_SVG`": converting to PNG (${width}px)..."
```

Comment:
Or, perhaps even better, factor it out to a variable so that the two occurrence always stay in sync, although it's unlikely we'll be changing this in the foreseeable future. Whichever you prefer.

Reply:
Fixed.